GNUmakefile

```diff
@@ -128,6 +128,10 @@ test_smoke_raw: test_need_operator_assets ## Run ALL integration smoke tests, on
 test_soak_ocr: test_need_operator_assets ## Run the OCR soak test
 	cd ./integration-tests && go test -v -run ^TestOCRSoak$$ ./soak -count=1 && cd ..
 
+.PHONY: test_soak_forwarder_ocr
+test_soak_forwarder_ocr: test_need_operator_assets ## Run the Forwarder OCR soak test
+	cd ./integration-tests && go test -v -run ^TestForwarderOCRSoak$$ ./soak -count=1 && cd ..
```

Comment:
Since this is the third similar instance, would it make sense to parameterize this as `make test_soak <xxx>`?
nit/ but related, consider naming these `TestSoak*` or `TestSoak_*` to align with the go testing hierarchy?

Reply:
oh, or ever drop `Soak` from the test name, since we are already in `package soak`?

Reply:
good idea, there is gonna be even more. I think I would put it for other tiny refactor ticket. Need to verify if those cmd's are/arent used in any ci/cd/scripts